releases/v1.1.0.toml

```diff
@@ -102,7 +102,7 @@ In terms of testing, we've passed:
 
 The containerd test coverage on GCE is equivalent with Docker now.
 
-All the test results are public: https://k8s-testgrid.appspot.com/sig-node-containerd.
+All the test results are public: https://prow.k8s.io/?repo=containerd%2Fcontainerd.
```

Comment:
Should we  edit an existing release file?

Reply:
Maybe we can add `release/` toml files to lycheeignore?

Reply:
> Maybe we can add release/ toml files to lycheeignore?

Fixed in 98526c9d3866cbc8f9c232e19488078ce0920c8e